modules/bootstrap/main.tf

```diff
@@ -1,3 +1,7 @@
+locals {
+  bootstrap_filenames = { for f in fileset(var.bootstrap_files, "**") : "${var.bootstrap_files}/${f}" => f }
+  filenames           = merge(local.bootstrap_filenames, var.files)
```

Comment:
We should create a merge function to compare values of these two maps (local.bootstrap_filenames, var.files), instead of their keys.